quota/etcd/quotaapi/quota_server.go

```diff
@@ -16,21 +16,54 @@
 package quotaapi
 
 import (
+	"github.com/coreos/etcd/clientv3"
 	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/google/trillian/quota/etcd/quotapb"
+	"github.com/google/trillian/quota/etcd/storage"
+	"github.com/google/trillian/quota/etcd/storagepb"
 	"golang.org/x/net/context"
+	"google.golang.org/genproto/protobuf/field_mask"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 var errNotImplemented = status.Error(codes.Unimplemented, "not implemented")
 
 // Server is a quotapb.QuotaServer implementation.
```

Comment:
I'll consider starting with the README next time. I didn't because I want to try it myself first and possibly tune the experience a bit before writing it down.

I expect it wasn't that hard to review, though.